roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -94,3 +95,13 @@ def exec_oc(execute_module=None, namespace="logging", cmd_str="", extra_args=Non
             raise OpenShiftCheckException(msg)
 
         return result.get("result", "")
+
+    def oc_cmd(self, cmd_str, extra_args, task_vars):
+        """Wrap parent exec_oc method. Allows for testing without actually invoking other modules."""
```

Comment:
This docstring is out-of-sync. `exec_oc` is not a parent method, it is defined right here in `LoggingCheck`, and not in the parent class, `OpenShiftCheck`.

Tip: document what it does and why someone would want to call this method and what to expect from the return value. No need to make references to class hierarchy nor to "testing without actually invoking other modules" (typically that is not interesting, you can always monkey-patch things in Python... one could monkey-patch `exec_oc` itself and this method wouldn't even be needed. It seems that its existence is solely to let you pass in 2 arguments less.

Suggestion:

```python
    def oc_cmd(self, cmd_str, extra_args, task_vars):
        """Same as exec_oc with a predefined execute_module and namespace."""
```

I almost wrote that this simplifies calls to `exec_oc`, but then I noted that `exec_oc` has all optional arguments while `oc_cmd` has 3 required args, so maybe it is not all that more convenient to call?! All calls I saw pass and empty `[]` to `extra_args`.